Documentation/UserTsconfig/Options.rst

```diff
@@ -847,6 +847,43 @@ pageTree.showPathAboveMounts
       :alt: Active user db mount
 
 
+.. index:: Password; Reset
+.. _useroptions-passwordReset:
+
+passwordReset
+-------------
+
+:aspect:`Datatype`
+   boolean
+
+:aspect:`Default`
+   true
+
+:aspect:`Description`
+   If set to `0` the initiating of the password reset in the backend
+   will be disabled. This does not affect the  the password reset by
+   cli command.
+
+   Disabling the password reset by user TSconfig overrules disabling the
```

Comment:
This paragraph is weird. Would enabling with user TSConfig overrule the global setting? Because in this example both disable the setting, which doesn't make much sense I guess.